recipes/coreutils/coreutils.inc

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "The GNU Core Utilities provide the basic file, shell and text \
+manipulation utilities. These are the core utilities which are expected to exist on \
+every system."
+LICENSE = "GPLv3+"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+
+inherit autotools gettext pkgconfig
+
+require conf/fetch/gnu.conf
+
+SRC_URI = "${GNU_MIRROR}/${PN}/${PN}-${PV}.tar.xz"
+
+EXTRA_OECONF:>native += " --without-gmp"
+EXTRA_OECONF:>machine += " --enable-install-program=arch --libexecdir=${libdir}"
```

Comment:
Why not `--libexecdir=${libexecdir}` ? (which is automatically added by autotools class btw.)